scripts/check-repo-names.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+SCRIPTNAME=${0##*/}
+
+rename_script="scripts/check-repo-names.sh"
```

Comment:
maybe you wanted to use `SCRIPTNAME` as an override for this?

Reply:
No, it was just leftover copypasta. Removed in 54d5cb818e2735bc18395a05bb33a0fa33585b2b